### PR TITLE
 DTS 2159378: Fix AggregateError

### PR DESCRIPTION
**Pull request by v-mohithgc**
**Task name**:  AKV1 and AKV2

**Description**:  DTS 2159378

**Attached related issue:** (Y/N) <Please add link to related issue here>

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

**Review by manolerazvan — APPROVED** [review]

**Review by kirill-ivlev — APPROVED** [review]

File: Tasks/AzureKeyVaultV1/task.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 1,
-    "Minor": 238,
-    "Patch": 2
+    "Minor": 239,
```
Comment (kirill-ivlev) [inline Tasks/AzureKeyVaultV1/task.json:17]: please bump task version to the current sprint

File: Tasks/AzureKeyVaultV1/task.loc.json [hunk 2]
```diff
@@ -14,8 +14,8 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 1,
-    "Minor": 238,
-    "Patch": 2
+    "Minor": 239,
```
Comment (kirill-ivlev) [inline Tasks/AzureKeyVaultV1/task.loc.json:17]: please bump task version to the current sprint

File: Tasks/AzureKeyVaultV2/task.json [hunk 3]
```diff
@@ -14,7 +14,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 2,
-    "Minor": 238,
+    "Minor": 239,
```
Comment (kirill-ivlev) [inline Tasks/AzureKeyVaultV2/task.json:17]: please bump task version to the current sprint

File: Tasks/AzureKeyVaultV2/task.loc.json [hunk 4]
```diff
@@ -14,7 +14,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 2,
-    "Minor": 238,
+    "Minor": 239,
```
Comment (kirill-ivlev) [inline Tasks/AzureKeyVaultV2/task.loc.json:17]: same

**Review by kirill-ivlev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**